src/it/resources/com/google/checkstyle/test/chapter4formatting/rule44columnlimit/InputColumnLimit.java

```diff
@@ -50,4 +51,55 @@ int test1(int badFormat1, int badFormat2, final int badFormat3) throws java.lang
 
   // Very long url with valid href: href    = "www.google.com/search?hl=en&q=java+style+guide+checkstyle+check+href+length+limit&btnG=Google+Search"
   int validHrefWithWhiteSpaces = 54;
+
+  // @cs-: LineLength for +1 lines
+  int aaaarealllllllllllllllllyyyyyyyyyyylllllllloooooooooooooooonnnnnnnnnnnnnnnnnggggggggggvvvvvvaarriaaabllee = 99; // ok
+
+  // @cs-: LineLength for +1 lines
+  void longggggggggggggggggggggmethoooooooooooooooooooooodddddddddddddddddddddddddddddddddddddd(int x, int y) {} // ok
+
+  // @cs-: LineLength for +1 lines
+  void testingParametersNames(int areallllllyyyyyyyyyyyyyyyyyylonnnnggggggggggname, int anotherlongnameblahblahblah) {} // ok
+  // Above code is wrap-able. User should not use suppression for such cases
+
+  // @cs-: LineLength for +1 lines
+  InputColumnLimit inputColumnLimit = (InputColumnLimit) ((Object) new com.google.checkstyle.test.chapter4formatting.rule44columnlimit.InputColumnLimit()); // ok
+  // Above code is wrap-able. User should not use suppression for such cases
+
+  void testing() {
+    try {
+      throwExceptionBasedOnCondition();
+    // @cs-: LineLength for +1 lines
+    } catch (IOException | NullPointerException | ArrayIndexOutOfBoundsException | ClassNotFoundException ex) {
+      // Above code is wrap-able. User should not use suppression for such cases
+      System.out.println(ex.getMessage());
+    }
+  }
+
+  // @cs-: LineLength for +2 lines
+  void throwExceptionBasedOnCondition()
+      throws IOException, NullPointerException, ArrayIndexOutOfBoundsException, ClassNotFoundException { // ok
```

Comment:
change all `// ok` to '// suppressed'
fyi: https://github.com/checkstyle/checkstyle/issues/13213

Reply:
> change all `// ok` to '// suppressed' fyi: #13213

done

https://github.com/checkstyle/checkstyle/blob/0386838241ac8d778427e82fb9e215a991ad6c4e/src/it/resources/com/google/checkstyle/test/chapter4formatting/rule44columnlimit/InputColumnLimit.java#L63-L65